Ginger/GingerCore/Actions/ActSwitchWindow.cs

```diff
@@ -86,10 +86,24 @@ public Int32 WaitTime
             {
                 string val = GetInputParamValue("WaitTime");
                 Int32 intVal = -1;
+
                 if (String.IsNullOrEmpty(val))
+                {
```

Comment:
i think we can simplify it like below instead of too many if conditions. Test it by loading old xml to ensure we are not changing the value.
Please add also 2 unit test if value is not null or empty then wait time should be equal to the value specified
If value is  null or empty then it should be equal to 30

 Int32 intVal = 30;
if (!String.IsNullOrEmpty(val))
{
   Int32.TryParse(val, out intVal);
}

Reply:
Pushed discussed changes and also added unit test